tests/cypress/support/accessibility.js

```diff
@@ -39,7 +39,7 @@ function getConfigurationForCLITests () {
 }
 function getConfigurationForGUITests () {
   return it('Check for possible accessibility errors at all logging levels set below in accordance with WCAG AA requirements', () => {
-    checkA11y({
+    checkA11y(null, null, {
```

Comment:
Passing `null, null, {...}` is wrong as the arrow function is declared (above in the same file) to take just one parameter called `options`. So this change should be reverted.

The reason that the table no longer displays has to do with the options object declared here which is somehow wrong, but when passed as third parameter it never gets used.